apis/filesystem/src/test/java/org/jclouds/filesystem/integration/FilesystemContainerIntegrationTest.java

```diff
@@ -196,6 +196,7 @@ public void testSetContainerAccess() throws Exception {
 
    @Override
    public void testClearWithOptions() throws InterruptedException {
-      throw new SkipException("filesystem does not support clear with options");
+//      throw new SkipException("filesystem does not support clear with options");
+      super.testClearWithOptions();
```

Comment:
Remove this method entirely?  The subclass method will get called automatically.

Reply:
Yeah it was there just for testing purposes to easily run that single test.